src/libs/OptionsListUtils.js

```diff
@@ -143,56 +147,52 @@ function addSMSDomainIfPhoneNumber(login) {
 }
 
 /**
- * Returns avatar data for a list of user logins
+ * Returns avatar data for a list of user accountIDs
  *
- * @param {Array<String>} logins
+ * @param {Array<Number>} accountIDs
  * @param {Object} personalDetails
  * @returns {Object}
  */
-function getAvatarsForLogins(logins, personalDetails) {
-    return _.map(logins, (login) => {
-        const userPersonalDetail = lodashGet(personalDetails, login, {login, avatar: ''});
+function getAvatarsForAccountIDs(accountIDs, personalDetails) {
+    return _.map(accountIDs, (accountID) => {
+        const userPersonalDetail = lodashGet(personalDetails, accountID, {login: '', accountID, avatar: ''});
         return {
-            source: UserUtils.getAvatar(userPersonalDetail.avatar, userPersonalDetail.login),
+            source: UserUtils.getAvatar(userPersonalDetail.avatar, userPersonalDetail.accountID),
             type: CONST.ICON_TYPE_AVATAR,
             name: userPersonalDetail.login,
         };
     });
 }
 
 /**
- * Returns the personal details for an array of logins
+ * Returns the personal details for an array of accountIDs
  *
- * @param {Array} logins
+ * @param {Array} accountIDs
  * @param {Object} personalDetails
  * @returns {Object} – keys of the object are emails, values are PersonalDetails objects.
  */
-function getPersonalDetailsForLogins(logins, personalDetails) {
-    const personalDetailsForLogins = {};
+function getPersonalDetailsForAccountIDs(accountIDs, personalDetails) {
+    const personalDetailsForAccountIDs = {};
     if (!personalDetails) {
-        return personalDetailsForLogins;
+        return personalDetailsForAccountIDs;
     }
-    _.chain(logins)
-
-        // Somehow it's possible for the logins coming from report.participants to contain undefined values so we use compact to remove them.
-        .compact()
```

Comment:
it looks like this applies to accountID's too - there was regression fixed by this PR: https://github.com/Expensify/App/pull/29930